github3/issues/issue.py

```diff
@@ -240,6 +242,13 @@ def is_closed(self):
             return True
         return False
 
+    def is_locked(self):
+        """Checks if the issue is locked.
+
+        :returns: bool
+        """
+        return self.locked
```

Comment:
Will this eventually have an API call? If not, I'd rather not have a method to return this attribute. `is_closed` is a little special mostly because the API can be inconsistent and you'd want to check either of those conditions.